config/toggles.js

```diff
@@ -126,6 +126,8 @@ const shortTermToggles = {
 
   returnUser: environmentVariableTrue(pe, 'RETURN_CREATEDBY_ON_TOKEN_INPUT'),
 
+  checkCacheOnPost: environmentVariableTrue(pe, 'CHECK_CACHE_ON_POST'),
+
```

Comment:
Short-term, yes, just so we can control when we enable it in prod, and can turn it off if there's anything amiss.